llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -1494,7 +1494,8 @@ def FeatureISAVersion9_5_Common : FeatureSet<
   [FeatureFP8Insts,
    FeatureFP8ConversionInsts,
    FeatureCvtFP8VOP1Bug,
-   FeatureGFX950Insts
+   FeatureGFX950Insts,
+   FeatureAddressableLocalMemorySize163840
```

Comment:
This means that gfx950 has both `FeatureAddressableLocalMemorySize65536` and `FeatureAddressableLocalMemorySize163840`, which seems like it should be disallowed in `GCNSubtarget::checkSubtargetFeatures`.

Reply:
It does end up taking the maximum, so it works